sw/device/lib/handler.c

```diff
@@ -13,31 +14,16 @@
  */
 static uint32_t get_mtval(void) {
   uint32_t mtval;
-  asm volatile("csrr %0, mtval" : "=r"(mtval) : :);
+  asm volatile("csrr %0, mtval" : "=r"(mtval));
```

Comment:
This should be a separate commit.

Reply:
Ok, I'll do those in a followup PR at some point.